src/ansi-c/c_typecheck_expr.cpp

```diff
@@ -2841,6 +2841,50 @@ exprt c_typecheck_baset::do_special_functions(
     overflow.type() = bool_typet{};
     return overflow;
   }
+  else if(identifier == CPROVER_PREFIX "enum_is_in_range")
+  {
+    // Check correct number of arguments
+    if(expr.arguments().size() != 1)
+    {
+      error().source_location = expr.source_location();
+      error() << "__enum_is_in_range expects one argument" << eom;
```

Comment:
It should be `CPROVER_PREFIX` "enum_is_in_range`

Reply:
Fixed in both places.